grammar.js

```diff
@@ -527,6 +524,10 @@ org_grammar = {
       seq($._text, token.immediate('_'), token.immediate(/[^\p{Z}]/))
     )),
 
+    contents: $ => seq(
+      repeat($._nl),
+      repeat1(seq(repeat1($._text), repeat1($._nl))),
+    ),
```

Comment:
If you change this bit to:
``` javascript
    contents: $ => seq(
      repeat($._text),
      sep1(repeat1($._nl), repeat1($._text)),
    ),
```
then we should be good to go.
I think this is inconsistent with `$.body`, because that one only starts at the actual text, but... that's fine.

Reply:
That fixed it.